### PR TITLE
Add ReadMe.md and make the text body optional

### PR DESCRIPTION
**Pull request by xhuberty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
...
+
+The `SwiftTwigMailTemplate` represents a mail template that can generate Swift mails.
+
+## Example
+
+Because we consider that an example is better than everything else...
+
+Start by creating your mail template. Your template should have two blocks:
+
+```
```
Comment (moufmouf) [inline README.md]: use ```twig to get syntax highlighting

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+{% block body_text %}
+    Body without HTML.
+{% endblock %}
+```
+
+Now, let's create a `SwiftTwigMailTemplate` instance. This object will generate a `SwiftMail` from the twig template.
+
+```
```
Comment (moufmouf) [inline README.md]: use ```php to get syntax highlighting

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
-TODO
\ No newline at end of file
+[![Build Status](https://travis-ci.org/thecodingmachine/swift-twig-mail-template.svg?branch=master)](https://travis-ci.org/thecodingmachine/swift-twig-mail-template)
+[![Coverage Status](https://coveralls.io/repos/thecodingmachine/swift-twig-mail-template/badge.svg?branch=master&service=github)](https://coveralls.io/github/thecodingmachine/swift-twig-mail-template?branch=master)
+
+
+# Swift Twig Mail Generator
+
+This package is a twig implementation of the [thecodingmachine/swift-mail-template-interface](https://github.com/thecodingmachine/swift-mail-template-interface).
```
Comment (moufmouf) [inline README.md]: Les gens ne sont pas forcément au courant de la "swift-mail-template-interface". Du coup, ça ne sert pas à grand chose de commencer par cette phrase. On pourrait peut être dire plutôt en premier:

> This package takes a Twig template and generates a Switft mail from this template.

**Comment by moufmouf** [comment]
Unit tests needs to be updated. Also, do not forget to run php-cs-fixer.

File: README.md [hunk 4]
```diff
@@ -1 +1,80 @@
...
+[![Build Status](https://travis-ci.org/thecodingmachine/swift-twig-mail-template.svg?branch=5.0)](https://travis-ci.org/thecodingmachine/swift-twig-mail-template)
+[![Coverage Status](https://coveralls.io/repos/thecodingmachine/swift-twig-mail-template/badge.svg?branch=5.0&service=github)](https://coveralls.io/github/thecodingmachine/swift-twig-mail-template?branch=5.0)
+[![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/thecodingmachine/swift-twig-mail-template/badges/quality-score.png?b=5.0)](https://scrutinizer-ci.com/g/thecodingmachine/swift-twig-mail-template?b=5.0)
+[![Code Coverage](https://scrutinizer-ci.com/g/thecodingmachine/swift-twig-mail-template/badges/coverage.png?branch=5.0)](https://scrutinizer-ci.com/g/thecodingmachine/swift-twig-mail-template?b=5.0)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
+[![Open Source Love](https://badges.frapsoft.com/os/mit/mit.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
+
+# Swift Twig Mail Generator
+
+This package takes a Twig template and generates a Switft mail from this template.
```
Comment (moufmouf) [inline README.md]: Typo: Switft => Swift

File: README.md [hunk 5]
```diff
@@ -1 +1,80 @@
...
+
+## Installation
+
+```
+composer require thecodingmachine/swift-twig-mail-template
+```
+
+Once installed, you can start creating instance of the `SwiftTwigMailTemplate` class.
```
Comment (moufmouf) [inline README.md]: you can start creating **an** instance

File: README.md [hunk 6]
```diff
@@ -1 +1,80 @@
...
+$swiftMail->setFrom('sender@example.com');
+$swiftMail->setTo('recipient@example.com');
+
+// We assume that $mailer is a valid Swift_Mailer instance
+$mailer->send($swiftMail);
+```
+
+### Going further
+
+The `SwiftTwigMailTemplate` class is design for dependency injection. You have the possibility to set a deeper configuration for your mail that's why you have access to a bunch of method for:
```
Comment (moufmouf) [inline README.md]: You could rather say:

The `SwiftTwigMailTemplate` class has been designed with dependency injection in mind. Instances can be easily put in your container for easy reuse. Furthermore, the `SwiftTwigMailTemplate` class shares a lot with the `Swift_Mail` class. You can:
- xxxx
- xxxx
- ....

**Comment by moufmouf** [comment]
Pretty cool. I've just added a few comments. We can merge once they are taken into account. :+1:

**Comment by marcteyssier** [comment]
Greetings! 

I'd like  to tell you  about my last  journey, it  was full of  adventures, please read more here <http://special.travelbluehorizons.com/e4qapebt>

Take care, m.teyssier

**Pull request merged**